_posts/2020-12-17-security-hub.md

```diff
@@ -0,0 +1,319 @@
+
+It's like a [linter](https://en.wikipedia.org/wiki/Lint_(software)), but for your infrastructure. This is important as we can't depend on linting CloudFormation (or CDK, or Terraform) templates, since not all projects use CloudFormation, and there isn't a security-focused linter available as far as I can tell.
+
+For engineers and administrators responsible for managing infrastructure, this is a welcome tool for helping to maintain consistent security best practice and for keeping infrastructure secure.
+
+Whilst it's becoming easier to manage multi-account environments, there are some challenges in the rollout of Security Hub that you'll want to keep in mind. In this article, we'll cover our experience, and tips for ensuring a smooth rollout and keeping it relevant for your organisation.
+
+
+## Multi-account rollout of Security Hub
+
+*Before we start, a warning; enabling Security Hub and compliance packs will cause a large number of AWS Config rules to execute for the first time across your accounts. If you have subscribed to AWS Config notifications (e.g. default AWS Control Tower setup), then you will receive a large influx of notifications during the rollout. We recommend you disable these subscriptions temporarily.*
+
+Also, as we recommend for all cloud services, please review the pricing pages for [AWS Security Hub](https://aws.amazon.com/security-hub/pricing/) and [AWS Config](https://aws.amazon.com/config/pricing/) to assess their suitability for your budget. Enabling AWS Config does cause charges for configuration items recorded on every resource change, so keep that in mind.
+
+Now, the first step to using Security Hub is to elect the admin account and then enable all other accounts as members. It consolidates findings into the admin account for each region to give a centralised view of your security state.
+
+The method we used to rollout Security Hub at the time was the [Security Hub Enabler](https://github.com/aws-samples/aws-control-tower-securityhub-enabler) tool provided by AWS. As of November 2020, Security Hub now supports [AWS Organizations](https://aws.amazon.com/organizations/), which is a much simpler method for enabling Security Hub.
+
+To [elect the Security Hub admin account](https://docs.aws.amazon.com/cli/latest/reference/securityhub/enable-organization-admin-account.html), run this via the AWS CLI for your Organization root account (where `123456789012` is the account ID you want to elect) in each of the regions you want to use Security Hub:
+
+```sh
+aws securityhub enable-organization-admin-account --admin-account-id 123456789012
```

Comment:
what would be the hot take for which account this goes in? e.g.
- if you are using organizations but not LZ/control tower... use account Y
- if you are using LZ / control tower, you might choose account X

My thought was, it goes in the security account, not the main account - but it's not really stated anywhere that I could see.

Reply:
Yeah, recommendation would be the security / audit account